### ISSUE TITLE
error: unknown option `hook'

### ISSUE THREAD

**Issue by alialaa** [commitizen/cz-cli#661]
Hello, I am trying to use 'git commit' using a husky hook as mentioned in the docs but I get ```error: unknown option `hook'``` when husky is trying to run ```exec < /dev/tty && git cz --hook```

**Comment by Indimeco** [comment]
@alialaa Had this issue. Updating commitizen to 4.0.3 fixed it for me.

**Comment by alialaa** [comment]
@Indimeco Are you sure? I am already using this version.

**Comment by Indimeco** [comment]
Yeah, pretty sure this is how I fixed it yesterday. To my understanding the error is saying "you supplied an argument to git-cz that does not exist" i.e., I was using a version where --hook haden't been implemented yet.

**Comment by vicentelyrio** [comment]
I had to rerun the commands `./node_modules/.bin/commitizen` and `./node_modules/.bin/commitizen init cz-conventional-changelog --save-dev --save-exact`

**Comment by alialaa** [comment]
@vicentelyrio thanks I did that and it worked

**Comment by DWboutin** [comment]
Thanks a lot @vicentelyrio

**Comment by gberghauserapotheke** [comment]
@vicentelyrio vlw meu amigo, to com mesmo problema

**Comment by not-a-large-language-model** [comment]
Adding some config to package.json was the fix for me! https://github.com/typicode/husky/issues/935#issuecomment-1741485415
